drivers/charger/charger_max20335.c

```diff
@@ -93,6 +116,37 @@ static int max20335_get_status(const struct device *dev, enum charger_status *st
 	return 0;
 }
 
+static int max20335_get_charger_online(const struct device *dev, enum charger_online *online)
+{
+	enum {
+		MAX20335_CHGIN_IN_NOT_PRESENT_OR_INVALID,
+		MAX20335_CHGIN_IN_PRESENT_AND_INVALID,
```

Comment:
Shouldn't it be `MAX20335_CHGIN_IN_PRESENT_AND_VALID` instead?

Reply:
You're right. Fixed.